File: src/main/java/jp/co/neut/ts/practice/entity/MstUser.java
package jp.co.neut.ts.practice.entity;

import org.seasar.doma.Column;
import org.seasar.doma.Entity;
import org.seasar.doma.Table;
import org.springframework.data.annotation.Id;

/**
 *
 */
@Entity
@Table(name = "MST_BOOK")
public class MstUser {

    /** */
    @Column(name = "BOOK_NAME")
    public String bookName;

    /** */
    @Id
    @Column(name = "BOOK_ID")
    public String bookId;

    /** */
    @Column(name = "BOOK_NO")
    public String bookNo;

    /** */
    @Column(name = "BOOK_AUTHOR")
    public String bookAuthor;

    /** */
    @Column(name = "BOOK_PUBLISHER")
    public String bookPublisher;

    /** */
    @Column(name = "BOOK_GENRE")
    public String bookGenre;

    /** */
    @Column(name = "BOOK_RELEASE")
    public String bookRelease;

    /** */
    @Column(name = "BOOK_BUY")
    public String bookBuy;

    /** */
    @Column(name = "BOOK_POSITION")
    public String bookPosition;

    /** */
    @Column(name = "BOOK_CHECK")
    public boolean bookCheck;

    /** */
    @Column(name = "BOOK_PICTER")
    public boolean bookPicter;

}

File: client/practice/src/app/dto/mst-user.ts
import { IReturnDto, ResultDto } from 'fl-framework-angular';
export class MstUser {
  bookId: string;
  bookName: string;
  bookNo: string;
  bookAuthor: string;
  bookPublisher: string;
  bookGenre: string;
  bookRelease: string;
  bookBuy: string;
  bookPosition: string;
  bookCheck: number;
  bookPicter:string;
}

export class UserParamDto {
  bookId: string;
  bookName: string;
  bookNo:string;
  bookAuthor: string;
  bookPublisher: string;
  bookGenre: string;
  bookRelease: string;
  bookBuy: string;
  bookPosition: string;
  bookCheck: number;
  bookPicter:string;
}

export class UserReturnDto implements IReturnDto {
  result: ResultDto;
  users: Array<MstUser>;
}

export class AllParamDto {
  bookId: string;
  bookName: string;
  bookNo: string;
  bookAuthor: string;
  bookPublisher: string;
  bookGenre: string;
  bookRelease: string;
  bookBuy: string;
  bookPosition: string;
  bookCheck: number;
  bookPicter:string;
}
export class AllReturnDto implements IReturnDto {
  result: ResultDto;
  users: AllParamDto;
  pass: AllParamDto;
}
